tests/kernel_bundle/kernels.h

```diff
@@ -0,0 +1,506 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides helper types and tool for tests on kernel bundle
+//
+*******************************************************************************/
+
+#ifndef __SYCLCTS_TESTS_KERNELS_H
+#define __SYCLCTS_TESTS_KERNELS_H
+
+#include "../../util/kernel_restrictions.h"
+#include "../common/common.h"
+#include <stdint.h>
+
+namespace kernels {
+using namespace sycl_cts;
+
+struct kernel_base {
+  // We are using unsigned long long to make it universal for all cases
+  // including atomic64
+  using element_type = unsigned long long;
+  using accessor_t = sycl::accessor<element_type, 1,
+                                    sycl::access_mode::read_write,
+                                    sycl::target::global_buffer>;
+  static constexpr element_type INIT_VAL = 1;
```

Comment:
The problem with uppercase elements is that it might conflict with macros.
It should not be a problem since we control the CTS but might be a problem with a broken SYCL implementation. Anyway, such an implementation would not deserve passing the CTS. ;-)

Reply:
The variables names was replaced into lowercase in this commit: 83e848f066993d91064182b48495c900047462dc